dace/cli/external_transformation_registry.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
+"""
+DaCe Transformation Repository Manager
+
+CLI tool for managing external transformation repositories.
+Allows users to register, clone, and manage transformation packages.
+"""
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import Dict, List, Optional
+from urllib.parse import urlparse
+
+
+class TransformationRepoManager:
+
+    def _set_path(self):
+        import dace
+        from dace import config
+        # Get default path for external transformations, eval $HOME if present
+        base_path_home_unevaluated_str = config.Config.get("external_transformations_path")
+        home_str = str(Path.home())
+        base_path_str = base_path_home_unevaluated_str.replace('$HOME', home_str)
```

Comment:
expand envvars like in dace/init or use the `dace.__external_transformations_path__` here (preferable).